aspire-orchestrator/README.md

```diff
@@ -1,54 +1,88 @@
+- [dotnet (9 or greater)](https://dotnet.microsoft.com/download)
+- [Python 3.8 or greater](https://www.python.org/downloads)
+- [NodeJs v18.12 or greater](https://nodejs.org/en/download)
 
 ## Run with .NET Aspire
 
-1. Make sure you have accepted the [dotnet trust dev certificate](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-9.0&tabs=visual-studio%2Clinux-sles#trust-the-aspnet-core-https-development-certificate)
-```bash
-dotnet dev-certs https --trust
-```
+1.  Make sure your workstation trusts
+    [ASP.NET Core HTTPS development certificate](https://learn.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?#trust-the-aspnet-core-https-development-certificate)
 
-2. Clone the repository
+        dotnet dev-certs https --trust
 
-```bash
-make
-cd aspire-orchestrator
-cd Aspire.AppHost
-dotnet run
-```
+2.  Clone the repository
+
+        make
+        cd aspire-orchestrator
+        cd Aspire.AppHost
+        dotnet run
 
 ## Deployment Steps with azd
 
-1. Clone the repository
-2. In the root folder, run
-```bash
-make
-```
-3. [Configure the app registration and configure workbenchapp and workbench service](../docs/CUSTOM_APP_REGISTRATION.md). Set the values in the `aspire-orchestrator/aspire-orchestrator/aspire-orchestrator/aspire-orchestrator/appsettings.json` file
-```json
-{
-    "EntraID": {
-        "ClientId": "<CLIENT_ID>",
-        "Authority": "https://login.microsoftonline.com/<TENANT_ID>"
-    }
-}
-```
-4. Authenticate with Azure Developer CLI
-```bash
-azd login
-```
-5. Create the infrastructure and deploy the application
-```bash
-azd up
-```
-6. Update the App Registration with the correct workbenchapp redirect URI
\ No newline at end of file
+1.  Clone the repository
+
+2.  [Configure the Entra app registration](../docs/CUSTOM_APP_REGISTRATION.md).
+    Copy the Entra **Application Id** and **Tenant Id** into
+    `aspire-orchestrator/Aspire.AppHost/appsettings.json` file.
+
+    If your Entra App allows both organizational and personal accounts
+    use `common` as Tenant Id, e.g. setting
+    `"Authority": "https://login.microsoftonline.com/common"`.
+
+        {
+          "EntraID": {
+            "ClientId": "<CLIENT_ID>",
+            "Authority": "https://login.microsoftonline.com/<TENANT_ID>"
+          }
+        }
+
+3.  In the root folder of the repository, run
+
+        make
+
+4.  Authenticate with Azure Developer CLI
+
+        azd login
+
+5.  Generate the Azure config files required in the next step
+
+        azd init --from-code --environment semanticworkbench
+
+6.  Create Azure resources and deploy the application
+
+        azd up
+
+    When asked for "**authority**", enter the same value set in appsettings.json.
+
+    When asked for "**clientId**", enter the same value set in appsettings.json.
+
+    These values are stored as Environment Variables and can be modified in Azure
```

Comment:
Same values as what? I presume TenantID for authority and ClientId for the other. So if this puts these into env vars should they start there instead of in appsettings? We've copied these values twice now, seems weird.

Reply:
During the deployment Aspire reads these values from `appsettings` and copies them into Azure Env Vars, inside the workbench resources (frontend and backend). The file itself is not deployed with the workbench.